src/components/tree/MainCardTop.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta, StoryObj } from '@storybook/react';
+import MainCardTop from './MainCardTop';
+
+const meta: Meta<typeof MainCardTop> = {
+  component: MainCardTop,
+  title: 'Tree/MainCardTop',
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * トップカード表示
+ */
+export const Default: Story = {
+  args: {},
+};
```

Comment:
↓動作確認がてら書いてみたけど、Storyの型部分がこんな感じの方が良いってことだよね？

```suggestion
import { Meta, StoryObj } from '@storybook/react';
import { ComponentProps } from 'react';
import MainCardTop from './MainCardTop';

const meta: Meta<typeof MainCardTop> = {
  component: MainCardTop,
  title: 'Tree/MainCardTop',
  tags: ['autodocs'],
};

export default meta;

type Story = StoryObj<typeof meta> & { args: ComponentProps<typeof MainCardTop> };

/**
 * トップカード表示
 */
export const Default: Story = {
  args: {},
};
```

Reply:
そう
修正した